pytorch_pfn_extras/training/extensions/lr_scheduler.py

```diff
@@ -56,14 +59,25 @@ def __init__(
         *,
         stepper: Any = _default_stepper,
         trigger: trigger_module.TriggerLike = (1, "epoch"),
+        wait_for_first_optimizer_step: bool = False,
         is_async: bool = True,
     ) -> None:
         self.scheduler = scheduler
         self.trigger = trigger_module.get_trigger(trigger)
         self.stepper = stepper
+        self.wait_for_first_optimizer_step = wait_for_first_optimizer_step
+        self._stepped = False
```

Comment:
I guess this flag is essentially unnecessary? Overhead of checking `self.scheduler.optimizer._step_count` for each `__call__` invocation sounds negilible to me.